CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# Contributing to Panda3D
+Thank you for looking into ways to contribute to Panda3D.
+Most information on how to contribute and get involved already exists in other places,
```

Comment:
This sentence is punctuated with a comma instead of a period.

Reply:
Fixed.